### ISSUE TITLE
Flatpak repository access

### ISSUE THREAD

**Issue by pemensik** [raceintospace/raceintospace#463]
Hi,

you might have noticed I pushed [flatpak](https://flatpak.org/) files into master. It is distribution independent way of shipping software on Linux machines. It would allow single binary to run on Ubuntu, Fedora, Debian or CentOS. Also few others.
I made a request to publish it on [flathub](https://flathub.org/apps/category/Game).

Its [review](https://github.com/flathub/flathub/pull/1822) seems to be successfully done. I was asked who should be given access. @peyre, @rnyoakum, are you interested?

Anyone else?

**Comment by pemensik** [comment]
My build was merged. If anyone wants access, please open issue in [flatpak repo](https://github.com/flathub/org.raceintospace.Raceintospace). It seems I have no admin privilege, because I don't see Settings tab like I do here. @peyre, can you add @pemensik to [raceintospace on flathub](https://github.com/flathub/org.raceintospace.Raceintospace)?

**Comment by peyre** [comment]
I'm sorry I've missed this all this time, Petr!  But I don't seem to have an option to manage users over there.  Might be I'm locked out too.

I was going to ask if we could modify what's there so the game identifies itself as version 1.2 instead of 2.0.0rc1, but I guess we don't have direct control of it.  I'll leave an Issue item for that.

Of course, if we could prepare a release, it would supersede this 1.2 version currently on Flatpak.

**Comment by pemensik** [comment]
Hmm, I have forgot how exactly is the flatpak processed. I am confident I am able to push into https://github.com/flathub/org.raceintospace.Raceintospace

Which should contain recipe for building shared flatpaks. Oh, default branch is somehow misleading. Branch org.raceintospace.Raceintospace does nothing. But branches beta and master catch changes and make it in builder.

**Comment by pemensik** [comment]
Oh, @peyre, have we lost domain raceintospace.org? It does not contain game resources anymore? Metadata info verification failed on my beta build: https://buildbot.flathub.org/#/builders/28/builds/12008

**Comment by peyre** [comment]
Yes, as I recall Michael asked me if I wanted to maintain it or if we should let it go.  We weren't really using it for much of anything as far as I could tell and I didn't have any plans for it, so I told him he could let it go.

**Comment by pemensik** [comment]
Well, I had used that domain in some files, because flatpak style uses reversed domains. I understand it costs money to keep that domain. I guess for a simple website we can use even github somehow. It should be possible to prepare *raceintospace.github.io* from repositories directly. It might be fine for showing manual for example, since you have spent quite a lot of time improving it.

**Comment by peyre** [comment]
This afternoon I received three emails from Flatpak.  I assume others are getting these as well?

org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.

whose subjects say:
runtime org.freedesktop.Platform branch 20.08 is end-of-life (Issue #3)
runtime org.freedesktop.Platform.VAAPI.Intel branch 20.08 is end-of-life (Issue #4)
runtime org.freedesktop.Platform.GL.default branch 20.08 is end-of-life (Issue #5)

**Comment by hweimer** [comment]
There is a pull request on the flathub repository (https://github.com/flathub/org.raceintospace.Raceintospace/pull/6) that is supposed to take care of these issues. I think @pemensik is the one who has to accept it.

**Comment by peyre** [comment]
What version number is it given?  The previous flatpak was confusingly called 2.0 as I recall.

**Comment by hweimer** [comment]
I'm not using flatpak, but as far as I can see it says 2.0.0rc3, i.e., the same as when building from source.
